examples/gitserver/hooks/detect-language

```diff
@@ -14,6 +14,11 @@ set -o pipefail
 function has {
   [[ -n $(git ls-tree --full-name --name-only HEAD ${@:1}) ]]
 }
+function hasglob {
+  git ls-tree --full-name --name-only HEAD | while read NAME; do
+    [[ "$NAME" == $1 ]] && return 0
```

Comment:
isn't this going to compare "*.csproj" with "somefile.csproj" and fail?  or is there some bash behavior here i'm not aware of?